src/mongo/db/mongod_main.cpp

```diff
@@ -1202,6 +1203,19 @@ ExitCode _initAndListen(ServiceContext* serviceContext, int listenPort) {
 
     OidcServerParameterLogger::log();
 
+    // Cannot use ServiceContext::ConstructorActionRegisterer to construct the
```

Comment:
I am not sure if that is the best way to initialize the registry while the periodic runner is already created in the service context.